### PR TITLE
docs: updates README.md

### PR DESCRIPTION
**Pull request by navarrothiago**
Signed-off-by: Thiago Navarro <navarro@accuknox.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
+### From Script
+
+kAmor has an installer script that will automatically grab the latest version of kAmor and install it locally.
+
 ```
 curl -sfL https://raw.githubusercontent.com/kubearmor/kubearmor-client/main/install.sh | sh
 ```
 
-To build and install, clone the repository and
+The binary will be installed in `<current_dir>/bin` folder.
```
Comment (daemon1024) [inline README.md:21]: Perhaps we should let the user know how to configure the path with `-s -- -b <path>/bin`

```sh
curl -sfL https://raw.githubusercontent.com/kubearmor/kubearmor-client/main/install.sh | sudo sh -s -- -b /usr/local/bin # Needed sudo cause /usr/local is priviledged path

# or 

curl -sfL https://raw.githubusercontent.com/kubearmor/kubearmor-client/main/install.sh | sudo sh -s -- -b $(go env GOPATH)/bin

```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
 restricts the behavior (such as process execution, file access, and networking
 operation) of containers at the system level.
 
 ## Installation
 
+The following sections show how to install the kAmor. It can be installed either from source, or from pre-built binary releases.
+
+### From Script
+
+kAmor has an installer script that will automatically grab the latest version of kAmor and install it locally.
```
Comment (daemon1024) [inline README.md:15]:
```suggestion
kArmor has an installer script that will automatically grab the latest version of kArmor and install it locally.
```

s/kAmor/kArmor across the changes 😅

**Review by daemon1024 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (daemon1024) [reply README.md:15]: Still a few `kAmor` --> `kArmor` changes needed 😅 😬

Comment (navarrothiago) [reply README.md]: Done

**Review by daemon1024 — APPROVED** [review]

**Pull request merged**